src/__tests__/pages/pupils/programmes/[programmesSlug]/options.test.tsx

```diff
@@ -52,6 +52,8 @@ describe("pages/pupils/programmes/[programmeSlug]/options", () => {
           programmes={programmes}
           baseSlug="physics-secondary-year-11"
           yearSlug="year-11"
+          examboards={[]}
+          tiers={[]}
```

Comment:
The PupilProgrammeListing now takes examboards and tiers meaning they can be calculated in the getStaticProps call.